lens/util/repo.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/filecoin-project/lotus/api"
 	"github.com/filecoin-project/lotus/blockstore"
 	"github.com/filecoin-project/lotus/build"
+	builtin "github.com/filecoin-project/lotus/chain/actors/builtin"
```

Comment:
This was the other fix. Lotus maintains this package to work with all versions of actor codes.